services/web/server/src/simcore_service_webserver/projects/_crud_api_read.py

```diff
@@ -47,7 +48,7 @@ async def _append_fields(
     return model_schema_cls.parse_obj(project).data(exclude_unset=True)
 
 
-async def list_projects(
+async def list_projects(  # pylint: disable=too-many-arguments
```

Comment:
maybe we should increase the number of arguments in pylint. but please not above 20